### DISCUSSION TITLE
Essentials &6default chat color code

### DISCUSSION THREAD

**Discussion by DaddyHunger** [EssentialsX/Essentials#4813] [Q&A] [answered]
Hey guys, so the MAJORITY of my server has been made custom in terms of chat - removing a lot of the default setups that plugins provide.

I can't seem to find a way to change the essentialsX chat responses to commands color it seems to be locked to &6 and &c and &4.
Is there a languages or config file that im blind to at the moment? or is it just not possible to edit the response outputs that essentials use?

**Comment by Chew** [comment, marked as answer]
This is described on our wiki: https://essentialsx.net/wiki/Locale.html
